docs/DeveloperGuide.md

```diff
@@ -135,13 +136,16 @@ and interprets the file to be loaded into the memory as a `TrackingList`.
 ![](images/storage_pathdir.png?raw=true "Storage Module file structure")
 
 `StoredTrackingList` is the `.txt` file stored in the same directory as the program.
-The location is determined by the `ConfigFile` class. 
+The location is determined by the `ConfigFile` class.
 
 `LogFile` takes the `TrackingList` every time the storage is updated
 and packages it into `json` formatted string.
 `LogFile` communicates with the disk by calling the methods
 `loadLogFile` and `saveLogFile` in the `StorageFile` class.
 
+`History` stores a record every time the `Command` `CheckIn` and `CheckOut` are successfully executed.
+It includes information about the `Person` details, `Location` and `Time` the person checked in. 
```

Comment:
It includes information about the person details, time and movement whether its in or out